bigquery/snippets/snippet_test.go

```diff
@@ -48,6 +48,10 @@ func TestAll(t *testing.T) {
 		t.Errorf("failed to create dataset: %v", err)
 	}
 
+	if err := updateDatasetAccessControl(client, datasetID); err != nil {
+		t.Errorf("Failed to update access control for dataset: %v", err)
```

Comment:
nit: lowercase f to be consistent.

but really the style can simply be:

`t.Errorf("updateDataSetAccessControl: %v", err)`

And you can add the datasetID if it's relevant.

`t.Errorf("updateDataSetAccessControl(%q): %v", datasetID, err)`

Reply:
If that's the dominant idiom, should I go ahead and plumb the method name into all the checks?

Reply:
if you're up for it! I'm happy to come and clean it up, too, if you don't have time.